src/transformers/models/encoder_decoder/modeling_tf_encoder_decoder.py

```diff
@@ -213,13 +213,6 @@ def dummy_inputs(self):
         # Add `decoder_input_ids` because `self.decoder` requires it.
         input_ids = tf.constant(DUMMY_INPUTS)
         dummy = {"input_ids": input_ids, "decoder_input_ids": input_ids}
-        # Add `encoder_hidden_states` to make the cross-attention layers' weights initialized
-        if self.config.add_cross_attention:
-            batch_size, seq_len = input_ids.shape
-            shape = (batch_size, seq_len) + (self.config.hidden_size,)
-            h = tf.random.uniform(shape=shape)
-            dummy["encoder_hidden_states"] = h
-
```

Comment:
Why is this part removed?

Reply:
`TFEncoderDecoderModel.call()` doesn't have `encoder_hidden_states` parameter, but `encoder_outputs`.
Moreover, `encoder_hidden_states` is always passed to the decoder with `encoder_hidden_states = encoder_outputs[0]`. Therefore, there is no need to add `encoder_hidden_states` in `dummy_inputs`.

(I don't remember why I did that before, probably in some intermediate commits, it was required)

Reply:
Looks good to me!